superbench/benchmarks/registry.py

```diff
@@ -105,32 +105,19 @@ def __get_benchmark_name(cls, benchmark_context):
         return benchmark_name
 
     @classmethod
-    def check_parameters(cls, benchmark_context):
-        """Check the validation of customized parameters.
+    def create_benchmark_context(cls, name, platform=Platform.CUDA, parameters='', framework=Framework.NONE):
```

Comment:
default is `Platform.CUDA` or `Platform.CPU`?

Reply:
Changed to Platform.CPU